neqo-transport/src/connection/mod.rs

```diff
@@ -525,8 +525,14 @@ impl Connection {
         qtrace!("[{self}] Retry CIDs: odcid={odcid} remote={remote_cid} retry={retry_cid}");
         // We advertise "our" choices in transport parameters.
         let local_tps = &mut self.tps.borrow_mut().local;
-        local_tps.set_bytes(tparams::ORIGINAL_DESTINATION_CONNECTION_ID, odcid.to_vec());
-        local_tps.set_bytes(tparams::RETRY_SOURCE_CONNECTION_ID, retry_cid.to_vec());
+        local_tps.set_bytes(
+            TransportParameterId::OriginalDestinationConnectionId,
```

Comment:
I wonder whether we should change the use statement at the top so that we can write: `OriginalDestinationConnectionId` rather than `TransportParameterId::OriginalDestinationConnectionId` (which takes a whole line on its own).